src/jdk.jdwp.agent/share/native/libjdwp/debugInit.c

```diff
@@ -396,6 +423,30 @@ cbEarlyVMInit(jvmtiEnv *jvmti_env, JNIEnv *env, jthread thread)
     LOG_MISC(("END cbEarlyVMInit"));
 }
 
+#if defined(J9VM_OPT_CRIU_SUPPORT)
+static void JNICALL
```

Comment:
I'm not sure `static` and `JNICALL` can be used together.

Reply:
The early callback functions here and normal callback functions in `eventHandler.c` all use this signature.